packages/discord.js/src/structures/BaseInteraction.js

```diff
@@ -268,12 +270,63 @@ class BaseInteraction extends Base {
     return this.type === InteractionType.MessageComponent && this.componentType === ComponentType.Button;
   }
 
+  // TODO: Replace with isAnySelectMenu in the next major - get rid of isAnySelectMenu
```

Comment:
Ngl I'd rather we removed this `isSelectMenu` and keep `isAnySelectMenu`

Reply:
But this would be a breaking change

Reply:
Not in this PR! In the future :D